queries/useGroupInviteQuery.ts

```diff
@@ -8,7 +8,11 @@ export const useGroupInviteQuery = (groupInviteId: string) => {
   const account = useCurrentAccount() as string;
 
   return useQuery<GroupInvite | undefined>({
+    // I need to learn more about how query keys work to determine
```

Comment:
Query keys are just a way to select a certain query status
Technically it can be anything, but it's normally best to combine the dependencies of the function

First we want to have a dependency on the group invite id, so if you look at a different group invite id, it will have a different state

Since the QueryClient is app wide, this means we should also set query keys by account since if you look at the group invite id as one account and then switch to another, it should be a completely different query

Reply:
That totally makes sense - thanks for the explanation. Went through `react-query` docs and they were super helpful, as well as the source. It's pretty clean!